src/yaml/aslYamlLanguageService.ts

```diff
@@ -249,12 +250,13 @@ function isChildOfStates(document: TextDocument, offset: number) {
     }
 
     languageService.format = function(
-        document: TextDocument
+        document: TextDocument,
+        range: Range,
```

Comment:
Is `range` used anywhere, just a placeholder argument i guess?

Reply:
placeholder. We might consider adding range based formatting in the future if there is need for it but I think for now, formatting the whole document should be sufficient.